cves/CVE-2017-11665.yml

```diff
@@ -93,16 +100,9 @@ fixes:
   note: ''
 - commit: ffcc82219cef0928bed2d558b19ef6ea35634130
   note: ''
-- commit: ''
-  note: ''
 vccs:
 - commit: 9fd6b8436008100699537c76e28c21a01e58cf4c
-  note: Identified by archeogit,Identified by SZZUnleashedIdentified by archeogit,Identified
```

Comment:
add a space after each comma

Reply:
This is on a removed line, I replaced it with an actual commit message